com.budgetwithbuckets.Buckets.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: com.budgetwithbuckets.Buckets
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: buckets.sh
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+modules:
+  - name: buckets
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 com.budgetwithbuckets.Buckets.desktop -t /app/share/applications
+      - install -Dm 644 com.budgetwithbuckets.Buckets.png -t /app/share/icons/hicolor/512x512/apps
+      - install -Dm 755 buckets.sh /app/bin/buckets.sh
```

Comment:
appdata is not being installed here

Reply:
Oh, I see what you mean now. Okay, pushed a change and now it should be installing in the right place.